EOSBasic/Plugins/UEOS/Source/UEOS/Public/Authentication/Authentication.h

```diff
@@ -86,18 +87,23 @@ struct UEOS_API FEpicAccountId
 	*/
 	FString						ToString() const;
 
-	/** 
+	/**
 	* Returns an Account ID from a String interpretation of one.
 	*
 	* @param AccountId the FString representation of an Account ID.
 	* @return FEpicAccountId An Account ID from the string, if valid.
 	*/
-	static FEpicAccountId		FromString( const FString& AccountId );
+	static FEpicAccountId		FromString(const FString& AccountId);
 
 	/** The EOS SDK matching Account Id. */
 	EOS_EpicAccountId			EpicAccountId;
+
+	UPROPERTY(BlueprintReadOnly, Category = "EpicAccountId")
+		FString DisplayName;
+	
```

Comment:
Not what this struct is for - see the sample. If we want an object to store more data about a user (like Display Name, Nickname, etc), then create a new struct.

Reply:
Yea, on my own project I made a new blueprintable struct - but saw that you had a change here in the past couple of days. Its a bit different now with multiple structs that are blueprintable - so will brainstorm myself.